bin/docker/run

```diff
@@ -5,12 +5,36 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 set -e
 
+# CLI handling
+
+pass_credentials_bintray=
+cmd=""
+
+while [ "$1" != "" ]; do
+    param="$1"
+
+    case $param in
+        --credentials-bintray)
+            pass_credentials_bintray=true
+            ;;
+        *)
+            cmd="$@"
+            break
+            ;;
+    esac
+
+    shift
+done
```

Comment:
No, you can always use `BEAVER_DOCKER_VARS` to pass whatever you want, is just a cosmetic change the one you are doing. Beaver is super flexible as it is, is just that you'll need to use uglier syntax for uncommon things and that was by design.